### ISSUE TITLE
Fix RollingSessionWindow error

### ISSUE THREAD

**Issue by gonzamontiel** [pendulum-chain/pendulum#17]
After upgrading to polkadot v.0.9.16, and plugin the new collator's runtime in Rococo, I observe the following error messages:

```
2022-02-11 16:22:01 [Relaychain] error=RollingSessionWindow(SessionsUnavailable { kind: RuntimeApi(Execution { runtime_api_name: "SessionInfo", source: FailedToDecodeReturnValue { function: "session_info", error: Error { cause: Some(Error { cause: Some(Error { cause: Some(Error { cause: Some(Error { cause: None, desc: "Not enough data to fill buffer" }), desc: "Could not decode `Public.0`" }), desc: "Could not decode `Public.0`" }), desc: "Could not decode `SessionInfo::validators`" }), desc: "Could not decode `Option::Some(T)`" } } }), info: Some(SessionsUnavailableInfo { window_start: 0, window_end: 0, block_hash: 0xaaf2cd1b74b5f726895921259421b534124726263982522174147046b8827897 }) })
2022-02-11 16:22:01 [Relaychain] Received msg before first active leaves update. This is not expected - message will be dropped. msg=ActiveDisputes(Sender { complete: false })
2022-02-11 16:22:01 [Relaychain] error=Sender(AskActiveDisputesCanceled) ctx="on FromOverseer"
2022-02-11 16:22:01 [Relaychain] Received msg before first active leaves update. This is not expected - message will be dropped. msg=ActiveDisputes(Sender { complete: false })
2022-02-11 16:22:01 [Relaychain] error=Sender(AskActiveDisputesCanceled) ctx="on FromOverseer"
```
```
2022-02-11 16:22:03 [Relaychain] Failed to request addresses of authorities: CallingRuntime(RuntimeApiError(Application(UnknownBlock("State already discarded for BlockId::Hash(0x19957b7151fdd00a875d05233d788703952481976fdb4948422e80509fe6c1ac)")))) 
```
Apparently as other users commented on Element we're not the only ones experiencing this. 
Testing branch for this issue is `legacy-bridge`.

**Comment by gonzamontiel** [comment]
Relevant related issue: https://github.com/paritytech/substrate/issues/10888

**Comment by gonzamontiel** [comment]
The Docker image we're using for running the node may have an out of sync clock. I am investigating if this is the cause of the error, and if this is also the cause of why the chain is not producing blocks at connecting to Rococo live.

**Comment by gonzamontiel** [comment]
Closing this since fixed in #18
